.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: Scala Merge CI
+
+on:
+  push:
+    branches: ['2.*.x']
+  pull_request:          ## TODO remove this line before merging
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build_and_test:
+    name: Build and Test
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+        java:
+          - 8
+          # - 11
+          # - 15
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # We don't use the setup-scala action because it installs the sbt-extras
+      # variant of the sbt script, which causes trouble with .jvmopts on Windows.
```

Comment:
Is this copy-pasted from somewhere else, or is this your text?  Is there more info I can look at about this?

Reply:
I wrote it as a note to myself, since the build was failing with `Invalid maximum heap size: -Xmx2G`. Seth saw the same failure on Travis: https://github.com/scala/scala/pull/9493#issuecomment-777145713

It was easy enough to try the official `sbt` script, since it is installed by default on the GitHub Actions runners, and the issue went away, so I added the note and moved on.

Now that you ask, I have looked into the failure.  Adding tracing to the sbt-extras `sbt` using `set -x` reveals the issue:

```
+ execRunner java $'-Xmx2G\r' $'-Xss1M\r' -jar /c/Users/runneradmin/.sbt/launchers/1.3.13/sbt-launch.jar setupPublishCore generateBuildCharacterPropertiesFile publishLocal
```

It's choking on the `'\r'` characters from the git checkout of the `.jvmopts` file on Windows using automatic end-of-line conversion.

The official `sbt` has handling to strip them:

```bash
loadConfigFile() {
  # Make sure the last line is read even if it doesn't have a terminating \n
  cat "$1" | sed $'/^\#/d;s/\r$//' | while read -r line || [[ -n "$line" ]]; do
    eval echo $line
  done
}
```

https://github.com/sbt/sbt-launcher-package/blob/635f05f2f38a166178fa18aeeb6a4e285bd595e4/src/universal/bin/sbt#L573

Reply:
Ah, I see.  Thanks for looking into that.  We could look to improving the sbt-extras script in general, but an easier option would be to fix it locally for .jvmopts with https://github.com/scalameta/metals/pull/1170/files.

Reply:
This was the issue which made me give up on programming, in the sense of straws and camels. Coming face-to-face with the fact that every program in the world needs to utilize some variation on "loadConfigFile" or it will be prone to losing the last line allowed me to realize that things will never be any good and I may as well not waste the rest of my life pretending otherwise. Just a little fun fact.

Reply:
re-iterating that once again the problem is caused by some program(er) (in this case git) that doesn't use Windows thinking "those weirdos on Windows like \r right? let's give them some \r's, I'm sure they'll appreciate that"

Reply:
Sounds like a reasonable default for beginners cloning repos and using notepad...  And it's configurable right?  Alas, the tyranny of the default means that it's probably more helpful for sbt-extras to handle this rather than making users to fixing their git attributes.

Reply:
Even notepad nowadays works fine with \n. Anyway, just lamenting.